examples/plotting/file/toolbar_autohide.py

```diff
@@ -1,3 +1,12 @@
+'''This example shows how to activate a toolbar that disappears if the cursor
+leaves a canvas.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.scatter, bokeh.models.tools.Toolbar.autohide
+    :refs: :ref:`reference` > :ref:`reference_models` > :ref:'reference_models_tools` > :ref:'reference_models_tools_Toolbar` > :ref:'reference_models_tools_Toolbar_autohide`, :ref:`first_steps` > :ref:`first_steps_first_steps_4` > `first_steps_first_steps_4_deactivating_and_hiding_the_toolbar`# noqa: E501
```

Comment:
Is this reference also working or dies this only work for 
```
:ref:`userguide_plotting` > :ref:`userguide_plotting_multi_lines`
```
?

Reply:
This works for any standard Sphinx reference that is defined in the ReST source for the docs and can be accessed via standard `:ref:`syntax. I don't know offhand if the ones above exists or not.